### DISCUSSION TITLE
Multiple Databases

### DISCUSSION THREAD

**Discussion by vzdesic** [oqtane/oqtane.framework#1196] [Q&A] [answered]
Please, I would like to better understand how we work with multiple databases. I understand that initial setup is such that what we call "master" database and tenant database is the same. There is ConnectionString named DefaultConnection defined in order to access that database. 

Now, how this should look like in case we want to separate "master" from tenant database or we introduce multiple tenants?
Do we introduce more ConnectionStrings to access all of them or some other way? 
How it is handled (access to the databases) in the source code?
Do we save Tenant related Settings in the database of that tenant?
Which mechanism enables us to fetch current Tenant Settings in the source code?
In some older videos on the web, I see - Tenant Menagement - section under Dashboard, now it is removed, why?

Please, also, add all what can be important to understand or is related to this topic!

Sorry for a long question. If you think that this is already explained somewhere else, refer to that, please. Thanks in advance!

**Comment by hishamco** [comment]
@vzdesic please checkout our discussion [here](https://github.com/oqtane/oqtane.framework/discussions/964)

**Comment by vzdesic** [comment, marked as answer]
Actually #731 is more related to my question. I found answers there. I see also that Tenant Management is moved to Site Management, and there is a option to create new Tenant.

All clear! No need for further explanations. Thanks!

**Reply by hishamco** [reply]
Ya the got merged, please mark it as answer and let the further details on the other discussion
